doc/src/manual/performance-tips.md

```diff
@@ -1101,6 +1101,17 @@ julia> @time begin
 Provided there is enough memory for the copies, the cost of copying the view to an array is
 far outweighed by the speed boost from doing the matrix multiplication on a contiguous array.
 
+## Consider StaticArrays for small fixed-size vector/matrix operations
```

Comment:
Maybe it makes sense to put out the `.jl` even in the title to highlight that we will talk about a package?